### ISSUE TITLE
Atlas name could be improved for anistropic atlases

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/bg-atlasgen#8]
At this line, the atlas name uses the first resolution value: https://github.com/brainglobe/bg-atlasgen/blob/f944baaf69004aca270c29dee50592369969db72/bg_atlasgen/wrapup.py#L111

This isn't particularly useful for non-isotropic atlases, but I'm not sure what would be better.

**Comment by vigji** [comment]
Yeah, the main reason for having the resolution in the name is mostly for atlases that come with different resolutions - eg, pretty useless for the zebrafish one. I guess that for a non-isotropic atlas that come in only one resolution it would not be very meaningful in general, as one atlas at multiple resolutions in my head make sense only if resampling is involved, at which point the atlas probably would not be anisotropic anymore

**Comment by adamltyson** [comment]
Makes sense to me. The resolution could be optional, thus only enforcing additional, resampled atlases to be isotropic? If we needed an additional, resampled anisotropic atlas, that could just have a different name.

**Comment by vigji** [comment]
yes, that would make sense for me!
